### DISCUSSION TITLE
When does or how can you rescan local file ID tags?

### DISCUSSION THREAD

**Discussion by OzGav** [music-assistant/support#527] [Q&A] [answered]
I have changed the ID tags for some of my local media files. Does anyone know how you can force MA to rescan the files to update the database? Or does this happen automatically?

**Comment by OzGav** [comment]
Actually I think I have found this. There is a RUN SYNC option in the menu in the very top right of the artists and abums pages.

**Comment by marcelveldt** [comment, marked as answer]
The scan runs automatically every 3 hours and at (re)load of the integration.
Besides that there's indeed a manual sync button tucked away in the top right menu.

**Reply by Gobbo28** [reply]
For me it would be a nice a feature, if I could trigger the sync from a Home Assistant Automation. I've got a task running which moves new media files directly to my NAS which is also included into Music Assistant. After that task I would like to trigger the sync.

**Reply by DarkFox** [reply]
I suppose I'll try my luck here, since my other thread got closed (https://github.com/orgs/music-assistant/discussions/1345), but since it's been almost a year, with no further discussion, I'm not getting my hopes up.

**Reply by marcelveldt** [reply]
One of the next releases will have a button entity in the HA integration, just a bit (more) patience...

**Reply by DarkFox** [reply]
That sounds great!
Sorry about the passive-agressiveness. I'm so used to being given the run-around, it's turned me cynical. :/

**Reply by OzGav** [reply]
There is now an option in settings>>core>>music controller to set the sync interval
